packages/core-data/src/selectors.ts

```diff
@@ -323,7 +332,10 @@ export const getEntityRecord = createSelector(
 			const fields = getNormalizedCommaSeparable( query._fields ) ?? [];
 			for ( let f = 0; f < fields.length; f++ ) {
 				const field = fields[ f ].split( '.' );
-				const value = get( item, field );
+				let value = { ...item };
```

Comment:
same cloning doesn't seem needed here?

Reply:
See my rationale above. Also, `item` is used later here so we definitely don't want to inadvertently mutate it.

Reply:
even if it's used later, value is never going to be equal to "item" after the loop

Reply:
👍 Removed in f00713e60d9a563b8b8349f81e394a33b0832c70